kie-dmn-feel/src/main/java/org/kie/dmn/feel/parser/feel11/FEELParser.java

```diff
@@ -38,10 +39,24 @@ public static FEEL_1_1Parser parse(FEELEventListenersManager eventsManager, Stri
 
         // pre-loads the parser with symbols
         defineVariables( inputVariableTypes, inputVariables, parser );
+        
+        parser.getHelper().dump();
+        
         return parser;
     }
 
     private static void defineVariables(Map<String, Type> inputVariableTypes, Map<String, Object> inputVariables, FEEL_1_1Parser parser) {
+
+        // switched order:
+        
+        inputVariableTypes.forEach( (name, type) -> {
```

Comment:
This is important, I believe during FEEL-compilation-stage the type is more important and shall be given precedence, so the order is inverted, also because... (below)